JSON.md

```diff
@@ -0,0 +1,53 @@
+Bencodex JSON Representation
+===========================
+
+## Introduction
+
+Bencodex JSON Representation is a standard way to represent a given Bencodex value to valid JSON text to be interchanged.
```

Comment:
To make it clear that the transformation is processed at data model, not parsing:

```suggestion
Bencodex JSON Representation is a standard way to represent a given Bencodex value to a valid JSON value to be interchanged.
```